api/package.json

```diff
@@ -1,11 +1,24 @@
 {
   "name": "lol-wrapper_api",
   "version": "0.0.1",
-  "main": "index.js",
+  "main": "dist/src/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "exit 0",
-    "test": "exit 0",
-    "serve": "exit 0"
+    "build": "tsc -p tsconfig.json",
```

Comment:
What are your thoughts on trying out newer stuff such as [swc](https://github.com/swc-project/swc) or [tsx](https://github.com/esbuild-kit/tsx)

Reply:
We can add a new story for this if we want. For now I think sticking with tsc is fine, especially as we get this story wrapped up